package.json

```diff
@@ -56,5 +56,8 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
+    },
+    "dependencies": {
+        "dtslint": "^4.2.1"
```

Comment:
Please submit this change as a separate PR if this was intentional

Reply:
Thanks @iisaduan, I've removed those extraneous changes.